InteractiveHtmlBom/ecad/genericjson.py

```diff
@@ -41,19 +40,14 @@ def parse(self):
         pcb = self.get_generic_json_pcb()
 
         if not self._verify(pcb):
-            self.logger.error('File ' + self.file_name +
-                              ' does not appear to be valid generic'
-                              ' interactivehtmlbom json file.')
+            self.logger.error('File %s does not appear to be valid generic'
+                              ' InteractiveHtmlBom json file.'
+                              .format(self.file_name))
```

Comment:
Sorry, I misremembered the syntax, correct substitution placeholder is `{}` not `%s`

Reply:
Well, shame on me too for not noticing that %s hadn't given the intended result before committing :)